husky_description/urdf/description.xacro

```diff
@@ -31,7 +31,7 @@ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSI
   <xacro:arg name="kinect_enabled" default="false" />
   <xacro:arg name="robotiq_grippers_enabled" default="$(optenv ROBOTIQ_GRIPPERS_ENABLED false)" />
  
-  <xacro:include filename="$(find husky_description)/urdf/husky.urdf.xacro" />
+  <xacro:include filename="$(find husky_description)/urdf/dual_arm_husky.urdf.xacro" />
```

Comment:
Hi @TheDash - the issue is that there is both a husky.urdf.xacro and a dual_arm_husky.urdf.xacro in this repository/branch and on the robot. The one that appears to be the most recent is the dual_arm_husky.urdf.xacro - when compiled with xacro it works. The one compiled from husky.urdf.xacro crashes. I believe a solution for this would be to either accept this change or to alternatively integrate dual_arm_husky.urdf.xacro into husky.urdf.xacro (cleaner).